srcmedia/scss/cards/_result.scss

```diff
@@ -21,19 +30,22 @@ ol.card-results {
 
     border-top: 0;
     width: $card-width;
-    /* space between cards: subtract three cards and divide remainder by 2 */;
-    margin-right: calc((100% - (#{$card-width} * 3)) / 2);
```

Comment:
rather than doing this calculation, for this pattern i think it's a lot easier to just use the `justify` and `align` values on the flex parent. we can remove all margins and size values from the cards themselves (except for bottom margin) and then just set on the container:

```css
justify-content: space-around;
align-items: center;
```

and 

```css
justify-content: space-between;
align-items: center;
```

on mobile.

this will automatically use whatever leftover space exists between the cards to space them out correctly.